packages/ui-components/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from '@storybook/addons';
+import { ComponentMeta } from '@storybook/react';
+import * as React from 'react';
+
+import { ThemeProvider } from '../..';
+import Pagination from './Pagination';
+
+export default {
+  title: 'UI Components/Pagination',
+  component: Pagination,
+} as ComponentMeta<typeof Pagination>;
+
+export const Default = () => {
```

Comment:
convert this one to args, see others stories for reference

Reply:
done